test/testenv/util.go

```diff
@@ -643,3 +643,13 @@ func CheckStringInSlice(stringSlice []string, compString string) bool {
 	}
 	return false
 }
+
+// GetConfigMap Gets the config map for a given k8 config map name
```

Comment:
Nit: As you are using code from feature-MC/CRD , would be great if you can simply import the whole file from source : https://github.com/splunk/splunk-operator/blob/feature-MC/CRD/test/testenv/util.go 
rather than one method. 
It will make it easier for rebasing when merging to develop.